sigar_linux_test.go

```diff
@@ -695,6 +695,7 @@ sl  local_address rem_address   st tx_queue rx_queue tr tm->when retrnsmt   uid
 			Expect(connList.List[0].RecvQueue).To(Equal(uint64(123)))
 			Expect(connList.List[0].Status).To(Equal(sigar.ConnStateListen))
 			Expect(connList.List[0].Proto).To(Equal(sigar.ConnProtoTcp))
+			Expect(connList.List[0].Inode).To(Equal(uint64(12095)))
```

Comment:
Took me a second to find these other `Inode` values, but now I see they're just not part of the diff: they were already in the test class :+1:

Reply:
Yep! They were zeroed-out in a couple of places and others were left intact as those lines were collected. The intact cases just got new asserts.